### PR TITLE
Feat: connect activities to backend

### LINKED ISSUE #19: Bug: requests never stop pending in the Network tab after refreshing the browser 5-8 times
The issue is detailed in [this card](https://trello.com/c/qt1F9MMI).

As I was adding screenshots on Trello, the solution suddenly came to mind: the connection pool is empty, so there must be a `client.release()` missing. And that was indeed the problem, after adding `client.release()`: 

image

Everything started to work as expected :sunglasses:.

### PR DESCRIPTION
**Pull request by Andrei0872**
_Refers to this [card](https://trello.com/c/96r6apAL)._

## Update

This also closes #19.

### CODE REVIEW

File: client/src/components/DailyPlanner.tsx [hunk 1]
```diff
@@ -226,13 +223,13 @@ function DailyPlanner (props: any, ref: any) {
 
     const activityId = ev.dataTransfer.getData('text');
     // TODO: adapt for Backend when the time comes.
```
Comment (DianaIfrosa) [inline client/src/components/DailyPlanner.tsx:225]: Non-serious review: "when the time comes" sounds so dramatic :))

**Comment by DianaIfrosa** [comment]
It looks good! Nice that you thought of a few use cases to prevent crashing. Maybe in the future we will need a "todo" component as well, for the "todo list"?

File: client/src/components/DailyPlanner.tsx (hunk 1 above)
Comment (Andrei0872) [reply client/src/components/DailyPlanner.tsx:225]: Yeah, things are so uncertain these days :laughing:.

**Comment by Andrei0872** [comment]
> Maybe in the future we will need a "todo" component as well, for the "todo list"?

@DianaIfrosa Sounds good! Maybe this we could add a _refactoring_ task for this, since the gist is [already here](https://github.com/Andrei0872/trip-designer/blob/master/client/src/components/Details.tsx#L118-L130), we'd just need to extract the _todo_-related part into its own component. What do you think?

**Review by Daria602 — APPROVED** [review]

**Pull request merged**